.tarpaulin.toml

```diff
@@ -1,7 +1,7 @@
 [all]
 exclude-files = [
     "tests/*",
-    "src/wrapped_radix_engine_toolkit/*",
```

Comment:
dont supress code coverage measurement of wrapped RET folder anymore

+DO suppress the code in bindgen, since it is not actual Sargon code.